src/robot/libraries/String.py

```diff
@@ -75,6 +75,37 @@ def convert_to_uppercase(self, string):
         | Should Be Equal | ${str2} | 1A2C3D |
         """
         return string.upper()
+    def convert_to_titlecase(self, string):
+        """Converts string to titlecase. It's preserve space also.
```

Comment:
This documentation should be enhanced a bit. I can do that myself if you aren't sure what to write.

Reply:
Followed comments from the **convert_to_lowercase**  and **convert_to_uppercase** keywords. I am little bit familiar with comments but your editing will help me lot to learn and add in other keywords what I am working on it. 

Thanks.